libs/features/personalization/personalization.js

```diff
@@ -427,12 +454,11 @@ export function handleCommands(commands, rootEl = document, forceInline = false)
       registerInBlockActions(cmd, manifestId, targetManifestId);
       return;
     }
-
+    const el = getSelectedElement({ selector, rootEl });
```

Comment:
Good call on pulling this out beforehand.  Makes me realize we should also do a return command.
```suggestion
    const el = getSelectedElement({ selector, rootEl });
    if (!el || (!(action in COMMANDS) && !(action in CREATE_CMDS)) return;
```
if we do that we can get rid of the else clause at the end.
could even do a return inside the first if clase and remove the else if wrap around the other commands